src/cockpit/389-console/src/lib/server/settings.jsx

```diff
@@ -270,6 +271,13 @@ export class ServerSettings extends React.Component {
                 valueErr = true;
                 disableSaveBtn = true;
             }
+            if (attr === 'nsslapd-disk-monitoring-threshold') {
+                const numVal = Number(value);
+                if (numVal < 4096) {
```

Comment:
It fails for me if I set 4096.
In the feature's code we have `if (*endp != '\0' || threshold <= 4096 || errno == ERANGE)` boundary check.

Reply:
This is fixed now in the core server